INSTALL.md

```diff
@@ -22,6 +22,7 @@ Required Software:
 * [DPC++ Compiler](https://www.intel.com/content/www/us/en/developer/tools/oneapi/dpc-compiler.html)
 * Microsoft Visual Studio\* (Windows\* only)
 * [MSYS2](http://msys2.github.io) (Windows\* only)
+* Python
```

Comment:
There's a line about it in the current master branch, two lines above where this would be inserted:
https://github.com/oneapi-src/oneDAL/blob/66f4858285c229a44be2ab17843edcbafaa989b9/INSTALL.md?plain=1#L23

It does mention "3.9" though so I guess there's no need to forego f-strings.

Reply:
Just for clarity, the mentioning of 3.9 was just introduced in your PR #2923 just after this PR was submitted (hence why I was confused). 3.9 requirement is set from here https://github.com/oneapi-src/oneDAL/pull/2923#discussion_r1782159936 .  I will rebase this PR to reflect your changes. 

The only python script that is run for make install is this python file and is the source of the dependency.  We do not test python consistently in any of our build steps. The private CI uses 3.12 as defined by infrastructure. The Ubuntu 24.04 docker runners use 3.12. The azure pipelines runners are also using default Ubuntu 22.04 python (3.10) runners. We do not test 3.9 nor 3.11 for oneDAL building.

Thus by changing these f-strings, we can support Ubuntu 20+ default python (i.e. python 2.7+).

@napetrov is there a reason why on build-side for oneDAL there is this python requirement?